src/StatisticCollector.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\SecretSanta;
+
+use Predis\Client;
+
+class StatisticCollector
+{
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    public function incrementUsageCount()
+    {
+        $currentYear = date('Y');
+        $currentMonth = date('m');
+
+        //If the key does not exist, it is set to 0 before performing the operation
+        $this->client->hincrby('date:' . $currentYear . '-' . $currentMonth, 'usageCount', 1);
```

Comment:
`"date: $currentYear - $currentMonth"` is more readable than concatenation  (same on next line)

Reply:
I made the change but I need that there is no space so it looks like ```"date:$currentYear-$currentMonth"```